x-pack/platform/plugins/shared/dataset_quality/common/translations.tsx

```diff
@@ -723,3 +718,32 @@ export const readLess = i18n.translate(
     defaultMessage: 'Read less',
   }
 );
+
+export const noFailureStoreAccessDescription = (
+  <FormattedMessage
+    id="xpack.datasetQuality.noFailureStoreAccess.description"
+    defaultMessage="{description}"
+    values={{
+      description: (
+        <FormattedMessage
+          id="xpack.datasetQuality.noFailureStoreAccess.warning"
+          defaultMessage="Documents that fail at ingestion are sent to the failure store. You don't have the required privileges to access failure store. Contact your administrator. {linkToDocs}"
+          values={{
+            linkToDocs: (
+              <EuiLink
+                external
+                target="_blank"
+                data-test-subj="datasetQualityFailureStoreDocsLink"
+                href="https://www.elastic.co/docs/solutions/observability/data-set-quality-monitoring#find-failed-documents-with-failure-store"
```

Comment:
Do we still use https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts?